rules/xcodeproj.bzl

```diff
@@ -845,18 +845,28 @@ def _add_pre_post_actions(target_name, scheme, key, actions):
             list.append({"script": action})
         payload[key] = list
 
+def _validate_output_path(output_path):
+    output_path_components = output_path.split("/")
+    prohibited_components = [".", ".."]
+    contains_dot_paths = any([elem in output_path_components for elem in prohibited_components])
+    if contains_dot_paths:
+        fail("The output_path provided cant contain the following path components: <%s>" % prohibited_components)
```

Comment:
does it mean the output has to be absolute path? if so i think it's worth calling out in the doc below?

Reply:
Yep. The reason for this is that we cant calculate the relative path to the sources. Same issue as in skylib `paths. relativize`, see: https://github.com/bazelbuild/bazel-skylib/blob/main/docs/paths_doc.md#pathsrelativize

> Because we do not have access to the underlying file system, this implementation differs slightly from Python's os.path.relpath in that it will fail if path is not beneath start

How would you be more explicit?

Reply:
sry i meant in the doc section `The output path to use when generating the xcode project` maybe update to `The output path to use when generating the xcode project, must be a relative path to workspace root`

Reply:
Done 🚀 

```
        The output path to use when generating the xcode project.
        Must be a relative path beneath the package where the xcodeproj rule is defined
```